create-page/create-page.js

```diff
@@ -0,0 +1,23 @@
+import { checkAuth, createPost } from '../fetch-utils.js';
+
+window.addEventListener('load', async () => {
+    await checkAuth();
```

Comment:
I believe this should be called outside of an event listener. I believe that's why I am not getting redirected when  I log in.